util/dvsim/BuildOnlyCfg.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
```

Comment:
No need for a shebang here, it's not a standalone script.

Reply:
I think this crept in by copy pasting the file. There are several other instances of this across our python files. Will clean up together with the header clean up above.